configs/example/gpufs/Disjoint_VIPER.py

```diff
@@ -107,8 +107,17 @@ def create(self, options, system, piobus, dma_devices):
         system.memories = cpu_abstract_mems
 
         gpu_abstract_mems = []
+
         for mem_ctrl in gpu_mem_ctrls:
-            gpu_abstract_mems.append(mem_ctrl.dram)
+            # memctrl
+            if hasattr(mem_ctrl, "dram"):
```

Comment:
instead of adding a second memory controller for the new HBM, shouldn't we add 2 dram's if the new model exists?  I guess this code seems a little complex to me, but maybe there is a good reason.  But it seems like this:

```
if (hbm_ctrl)
  gpu_abstract_mems.append(mem_ctrl.dram)
  gpu_abstract_mems.append(mem_ctrl.dram_2)
else if (!hbm_ctrl but using a different hbm)
  gpu_abstract_mems.append(mem_ctrl.dram)
else
  gpu_abstract_mems.append(mem_ctrl)
```
  
is more straightforward, no?

Reply:
ok, we can leave it be then